site/payment-gateway/integration.md

```diff
@@ -17,3 +34,48 @@ You need to contact your Holiday Extras representative to get your authenticatio
 ### 2. Ask for a channel
 
 When integrating with the Payment Gateway, we will provide you with a channel to pass in the body of every request. This is for us to use to determine who is using our Payment Gateway. Please see the endpoint documentation for usage on the `channel` parameter.
+
+### 3. Integrate the frontend payment script
+
+To integrate the script into your page you need a few prerequisites. You will need to:
+
+- have your domain served on https
+- add an empty placeholder HTML element into your form
+- load the payment script on the page
+- configure the script on page load
+
+Here is a very simple example snippet that demonstrates this:
+
+```html
+<form>
+  ... the contents of your existing form
+  <!-- Add the payment placeholder -->
+  <div id="hx-payment-placeholder"></div>
+  ... rest of your form with any other inputs and the submit button, etc
+</form>
+
+... any other content you have on your page
+
+<!-- Add the frontend payment script -->
+<script src="https://api.holidayextras.co.uk/payment-gateway/payment.1.0.0.min.js"></script>
+<!-- Configure the script -->
+<script>
+window.hxPayment.configure({
+  paymentFailed: false // you will need to dynamically provide the value for this
+})
+</script>
+```
+
+The placeholder div is used by the payment script to put the appropriate DOM elements it needs in order to get the customer card details.
+
+The fields that get created are:
+- expiry date input (needed for tokenising the card number)
+- token input (needed so that it gets submitted with your form to make a payment)
```

Comment:
It might be helpful to add the `name` attribute of the token field just to make it as easy as possible? This is what the partners will be looking out for when submitting the form so it would help identifying fields to use

Reply:
That's a good shout. I just had a thought that maybe we should add support for providing a custom token input field name in the options when the payment script gets configured. This would make sense for partners who already have some `token` field in their form. We want to have a small footprint, as much as possible so that they don't need to make a lot of changes in the end, only the bare minimum.